src/components/dashboard/FeedItems/ListEventItem.js

```diff
@@ -53,47 +59,65 @@ const InviteItem = ({ item, theme }) => (
   </Section.Row>
 )
 
-const NewsItem: React.FC = ({ item, eventSettings, styles }) => (
-  <View style={styles.rowContent}>
-    <FeedListItemLeftBorder style={styles.rowContentBorder} color={eventSettings.color} />
+const NewsItem: React.FC = ({ item, eventSettings, styles }) => {
+  const {
+    data: { sponsoredLink, sponsoredLogo },
+  } = item
+  const onSponsorPress = useOnPress(() => {
+    fireEvent(GOTO_SPONSOR)
```

Comment:
add link as parameter

Reply:
also, if there's event for **post** click, don't forget to send post's link with the event too